qwoyn/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyn",
+  "status": "live",
+  "network_type": "testnet",
```

Comment:
Assuming you mean to register a testnet and mainnet at the same time, then this should be mainnet

Reply:
based on other things I see, I don't think you meant to register a mainnet... so perhaps the mainnet files aught to be removed?